pom.xml

```diff
@@ -359,6 +359,12 @@
             </exclusions>
         </dependency>
 
+        <!-- Spring Core uses Commons Logging directly, causing core classes to bypass Logback configuration -->
+        <dependency>
+            <groupId>org.slf4j</groupId>
+            <artifactId>jcl-over-slf4j</artifactId>
+        </dependency>
+
```

Comment:
We use different logging configuration than what comes with tiamat, and turns out this dependency wasn't included. This is a bit of a smell IMHO, but since the logging system we use behaved a bit nicer with it, this is included in the PR.

Reply:
I'm not really sure what this means - I need more context.

Reply:
I'm going to drop this commit, it had an effect on our side (effectively discarded some jcl based internal logging freom core Spring classes), but the effect is neglible and not really important to us after all.